src/pages/Login.js

```diff
@@ -36,7 +37,7 @@ const Login = () => {
       const response = await baseURL.post("login", {
         userData: user
       })
-      console.log("Data", response.data);
+      handleLogin(response.data);
     } catch (error) {
       catchErrors(error, setError);
     } finally {
```

Comment:
Small comment on line 66 on the button element. You are using the ( || ) OR operator to disable the button until all inputs have been filled in. However, the loading variable is only set inside the handleOnsubmit function after the submit button is clicked. Therefore, it will never be set to true before submission. In my opinion you should remove the leading variable and just keep disabled like so disabled={disabled}. that should do it.

Reply:
No the button will be disabled either when not all fields are filled out or when all fields are filled out and you press it once, it will be loading and you can't press it again.